.coveragerc

```
@@ -0,0 +1,17 @@
+[run]
+data_file = coverage/.coverage
+branch = true
+
+[paths]
+source = fairtally/
+
+[html]
+directory = coverage/htmlcov/
+
+[xml]
+output = coverage/coverage.xml
+
+[report]
+omit =
+    venv*
+    tests/*
```

My virtualenv is called `env/` similar to what https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment guide does. 

Can we loosen the omit regex to
```suggestion
omit =
    *env*
    tests/*
```


Or instead use `include` instead of `omit`.
```suggestion
include=fairtally
```